app/models/transformation_mapping_item.rb

```diff
@@ -24,4 +31,33 @@ def destination_cluster
       errors.add(:destination, "EMS type of destination cluster or cloud tenant must be in: #{destination_types}")
     end
   end
+
+  def source_datastore
+    tm                   = TransformationMapping.find(self.transformation_mapping_id)
```

Comment:
Anywhere in this instance you can call `transformation_mapping` and it will return you the transformation mapping.  The `find` in unnecessary because of the `belongs_to` relationship https://github.com/ManageIQ/manageiq/pull/18835/files#diff-d2efbe990f72932d8117a6ce41b23ac0R2

Reply:
Done.